openfermioncirq/gates/common_gates.py

```diff
@@ -85,72 +92,26 @@ def __repr__(self) -> str:
         return '(ofc.FSWAP**{!r})'.format(self.exponent)
 
 
-class XXYYGate(cirq.EigenGate,
-               cirq.InterchangeableQubitsGate,
-               cirq.TwoQubitGate):
+class XXYYPowGate(cirq.EigenGate,
+                  cirq.InterchangeableQubitsGate,
+                  cirq.TwoQubitGate):
     """XX + YY interaction.
 
-    There are two ways to instantiate this gate.
-
-    The first is to provide an angle in units of either half-turns,
-    radians, or degrees. In this case, the gate's matrix is defined
-    as follows::
-
-        XXYY**h ≡ exp(-i π h (X⊗X + Y⊗Y) / 4)
-                ≡ exp(-i rads (X⊗X + Y⊗Y) / 4)
-                ≡ exp(-i π (degs / 180) (X⊗X + Y⊗Y) / 4)
-                ≡ [1 0             0             0]
-                  [0 cos(π·h/2)    -i·sin(π·h/2) 0]
-                  [0 -i·sin(π·h/2) cos(π·h/2)    0]
-                  [0 0             0             1]
-
-    where h is the angle in half-turns.
-
-    The second way is to provide a duration of time. In this case, the gate
-    implements the unitary::
-
-        exp(-i t (X⊗X + Y⊗Y) / 2) ≡ [1 0         0         0]
-                                    [0 cos(t)    -i·sin(t) 0]
-                                    [0 -i·sin(t) cos(t)    0]
-                                    [0 0         0         1]
-
-    where t is the duration. This corresponds to evolving under the
-    Hamiltonian (X⊗X + Y⊗Y) / 2 for that duration of time.
+    When exponent=1, swaps the two qubits and phases |01⟩ and |10⟩ by -i. More
```

Comment:
Is this just the ISWAP gate but with the opposite sign convention on the `i`? Maybe we should merge them.

Reply:
On second thought, there is some code which uses this class because it can't use Rxxyy because Rxxyy can't do Symbols. Let's merge them in a separate PR.